focus-android/app/src/main/java/org/mozilla/focus/cfr/CfrMiddleware.kt

```diff
@@ -40,13 +45,27 @@ class CfrMiddleware(private val appContext: Context) : Middleware<BrowserState,
             }
 
             next(action)
-
+            showCookieBannerCfr(context)
             showTrackingProtectionCfr(action, context)
```

Comment:
Here we are checking the `ContentAction.UpdateSecurityInfoAction` with is not relate to cookie banners, if you would like to observe an action that is related to cookie banner check the `CookieBannerAction.UpdateStatusAction`